app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -129,10 +129,6 @@ public function getCartDataJs($multipage = true)
             $hintData = $this->getAddressHints($customerSession, $sessionQuote);
             ///////////////////////////////////////////////////////////////
 
-            $orderCreationResponse = '';
```

Comment:
Actually I prefer to keep these variable definitions (as opposed to define this inside nested code below and use outside of the nest level)

Reply:
I have moved this before the try block below.